src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  background-color: $white-color;
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+  padding: 32px 0 16px;
+  padding-inline: 16px;
+  width: $card-width;
+
+  transition:
+    transform 300ms,
+    color 300ms,
+    background 300ms;
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The scale transformation on hover is set to 120%, which might be too large and could affect the layout. Consider using a smaller scale factor to ensure the card remains visually appealing and doesn't disrupt the surrounding layout.